src/components/Products/Filters.jsx

```diff
@@ -1,27 +1,50 @@
 import React from "react";
-import { Link, useHistory } from "react-router-dom";
+import { Link, useHistory, useLocation } from "react-router-dom";
 
 import FilterIcon from "../../assets/icons/filtro.png";
 
 const Filters = ({
-  categoryClicked,
-  categories: departments,
-  currentFilter,
+  departments,
+  types,
+  selectedFilter,
   setFilter,
+  setSelectedDepartment,
+  setSelectedTypes
 }) => {
+  const { search } = useLocation();
   const { push } = useHistory();
 
+  const updateQueryParams = (key, value) => {
+    const searchParams = new URLSearchParams(search);
```

Comment:
What if is empty? What about to add a default?

Reply:
Is about URLSearchParams? Well, if URLSearchParams is empty just create a new object URLSearchParams, then add a specified key/value pair as a new search parameter.
I'm not sure what I should add by default.